[CBRD-23886] Added File for New JDBC Repository and build

hwany7seo opened the pull request:
http://jira.cubrid.org/browse/CBRD-23886

hwany7seo committed:
[CBRD-23886] Added File for New JDBC Repository and build
http://jira.cubrid.org/browse/CBRD-23886

kisoo-han commented:
* build.sh, build.bat
  * Need to check for wrong option.
  * Need usage for help option
  * Check whether build.bat works in windows build

kisoo-han reviewed (changes requested):
* build.sh, build.bat
  * Need to check for wrong option.
  * Need usage for help option
  * Check whether build.bat works in windows build

hwany7seo committed:
[CBRD-23886] Fix build script and Change files permission
http://jira.cubrid.org/browse/CBRD-23886
 - Added check for wrong option.
 - Added usage for help option.
 - Check build work in build (windows, linux).

kisoo-han commented on build.bat:
```diff
@@ -0,0 +1,73 @@
+@echo. OPTIONS
+@echo.  clean			   Clean (jar and build file)
+@echo.  /help /h /?        Display this help message and exit
+@echo. Examples:
```
Please add 1 blank line before '**Examples**', and align each description in same  position.

kisoo-han commented on build.bat:
```diff
@@ -0,0 +1,73 @@
+
+:SHOW_USAGE
+@echo.Usage: %0 [OPTION]
+@echo.Build scrtip for CUBRID JDBC Driver
```
@echo.Build script for CUBRID JDBC Driver

kisoo-han commented on build.bat:
```diff
@@ -0,0 +1,73 @@
+@echo.  clean			   Clean (jar and build file)
+@echo.  /help /h /?        Display this help message and exit
+@echo. Examples:
+@echo.  %0             # JDBC Build
```
\# Build JDBC

kisoo-han commented on COPYING:
```diff
@@ -0,0 +1,193 @@
+Copyright (C) 2008 Search Solution Corporation. 
+Copyright (C) 2016 CUBRID Corporation.
+CUBRID is registered trademark of CUBRID Corporation.
```
Replace above 3 lines to
**Copyright (C) 2016 CUBRID Corporation.**

kisoo-han commented on README.md:
````diff
@@ -0,0 +1,50 @@
+  ```
+  ./build.sh 
+  ./build.sh claen
+  ```
````
./build.sh **clean**

hwany7seo committed:
[CBRD-23886] modify copyright and usage description (spell, align...)
http://jira.cubrid.org/browse/CBRD-23886

kisoo-han commented on CREDITS:
```diff
@@ -0,0 +1,16 @@
+* 1day1_org
+* shriekout
+* quartex
+* nori
```
add two contributors:
* eseokoh
* ribram

kisoo-han commented on README.md:
````diff
@@ -0,0 +1,50 @@
+- Windows
+  ```
+  build.bat 
+  build.bat clean 
````
delete:
build.bat clean

kisoo-han commented on README.md:
````diff
@@ -0,0 +1,50 @@
+- Linux
+
+  ```
+  ./build.sh 
````
sh build.sh
delete line './build.sh clean'

kisoo-han commented on README.md:
````diff
@@ -0,0 +1,50 @@
+  ./build.sh clean
+  ```
+
+### CUBRID JDBC Driver Installation Guide for Windows/Linux Users
````
change:
### CUBRID JDBC Driver

kisoo-han commented on README.md:
````diff
@@ -0,0 +1,50 @@
+  ```
+
+### CUBRID JDBC Driver Installation Guide for Windows/Linux Users
+- CUBRID JDBC driver (JAR File) implements an interface to enable access 
````
- CUBRID JDBC driver (JAR File) implements an interface to enable access 
+ implements an interface to enable access

kisoo-han commented on VERSION:
```diff
@@ -0,0 +1 @@
+11.0.1.0001
```
11.0.0.0249

kisoo-han commented on build.bat:
```diff
@@ -0,0 +1,85 @@
+)
+
+if "%ANT_PATH%" == "" (
+  echo "Please check ANT_HOME or ANT(excute file)"
```
- echo "Please check ANT_HOME or ANT(excute file)"
+ echo "Please check ANT_HOME or ANT(execute file)"

kisoo-han commented on README.md:
````diff
@@ -0,0 +1,50 @@
+### CUBRID JDBC Driver Source Build Guide (ANT Build Project)
+- Windows
+  ```
+  build.bat 
````
- build.bat
+ build

kisoo-han reviewed (approved):
Thank you for your efforts, no more comment. Looks good for me.

hwany7seo committed:
[CBRD-23886] added contributors in CREDITS file and modify script&README description (spell...)
http://jira.cubrid.org/browse/CBRD-23886

hwany7seo committed:
[CBRD-23886] Change new repo version
http://jira.cubrid.org/browse/CBRD-23886
